provisioner/utils/setup-env-debian.sh

```diff
@@ -43,3 +43,9 @@ else
     echo "local apt = $enable_local_repo ; NOT Enabling local apt. Packages will be pulled from remote..."
 fi
 
+# BIGTOP-4078 fix the solr smoketest need rm command
```

Comment:
You only need to add comment like this if the intent is unclear and nonintuitive and someone tend to think the code is outdated and unnecessary.